src/Apps/ViewingRoom/Routes/Works/__tests__/ViewingRoomWorksRoute.jest.tsx

```diff
@@ -167,7 +167,6 @@ const ViewingRoomWorksRouteFixture: ViewingRoomWorksRoute_Test_Query$rawResponse
             date: "2015",
             additionalInformation: "some description",
             href: "/artwork/bill-miles-beep-beep",
-            artistNames: "Artist Name",
```

Comment:
not sure why the typescript compiler started having issues with all these fields...